signer/src/testing/storage/postgres.rs

```diff
@@ -38,4 +39,29 @@ impl PgStore {
         .await
         .map_err(Error::SqlxQuery)
     }
+
+    /// Get all key-rotation events that have been logged in the database for the
+    /// given aggregate key.
```

Comment:
nit: can we add to the docstring that this doesn't check for canonical events only, but includes even forked events?

Reply:
Removed the fn in [b11e1b4](https://github.com/stacks-sbtc/sbtc/pull/1726/commits/b11e1b4b1b1fc89cd755600c4070d778929af7d8)